javascript/javascript1/week3/series-duration-of-my-life.js

```diff
@@ -0,0 +1,28 @@
+const TvSeriesDurations = [
+  { title: "One planet", days: 10, hours: 22, minutes: 12 },
+  { title: "Seven Worlds—One Planet", days: 21, hours: 4, minutes: 58 },
+];
+function logOutTvSeriesText() {
+  const averageLifeSpan = 80;
```

Comment:
good idea to place all your consts outside of function and on top of file. 
It's not wrong what you have :) it just makes it easier to read, tip for next time.